cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -1326,6 +1326,23 @@ pPoolStakeVerificationKeyFile =
          )
     )
 
+pStakePoolVerificationKeyHash :: Parser (Hash StakePoolKey)
+pStakePoolVerificationKeyHash =
+    Opt.option
+      (Opt.maybeReader spvkHash)
+        (  Opt.long "stake-pool-verification-key-hash"
+        <> Opt.metavar "HASH"
+        <> Opt.help "Stake pool verification key hash (hex-encoded)."
+        )
+  where
+    spvkHash :: String -> Maybe (Hash StakePoolKey)
+    spvkHash = deserialiseFromRawBytesHex (AsHash AsStakePoolKey) . BSC.pack
```

Comment:
We can do better than this now! We can use the bech32 support.

Which reminds me, Matthias had the excellent suggestion that where we accept bech32 we should to ease transitions and for compatibility, accept either bech32 or raw hex. I think we can do this straightforwardly in the api since our bech32 class is built on top of our raw serialisation class, so we could make a function to accept either.

So yes this is fine for now, but this should be an easy enhancement to add in. We could support file or bech32 in many places actually, various keys or key hashes. This draft CIP has the current list
https://github.com/cardano-foundation/CIPs/pull/6

Reply:
Gotcha. So I'll merge this and then revise it later.